README.md

```diff
@@ -182,17 +197,22 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://christmas.wannathis.one/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184478914-e0edf39e-aee6-46c9-a953-00749d471cc5.png' alt="Christmas 3D_free_illustrations" /></a> | [Christmas 3D](https://christmas.wannathis.one/)  <br><br> ✔ Free <br><br>Format <br>• PNG<br><br> ✌️Author <br>Wannathis                                                                                              |
 | <a href="https://drawkit.com/product/chubbs-character-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193478987-57701db5-dc94-4adc-af94-6d849b9beb8a.png' alt="Chubbs_Character_free_illustrations" /></a> | [Chubbs Character](https://drawkit.com/product/chubbs-character-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |
 | <a href="https://craftwork.gumroad.com/l/ZWiR"><img width='700' src='https://user-images.githubusercontent.com/74038190/184810212-6b2ee06f-81a7-413c-b967-a899e9b7b56c.png' alt="Control_free_illustrations" /></a> | [Control](https://craftwork.gumroad.com/l/ZWiR)  <br><br> ✔ Free <br><br>Format <br> • PNG<br> • Adobe Illustrator<br><br> ✌️Author <br> Craftwork.design                                                                                             |
+| <a href="https://drawkit.com/product/covid-19"><img width='700' src='https://user-images.githubusercontent.com/34191321/194713789-cd2508a7-a719-4bb6-90c6-18e6920c6bab.png' alt="COVID-19_icons_free_illustrations" /></a> | [COVID-19 Icons](https://drawkit.com/product/covid-19)  <br><br> ✔ Free <br><br>Format <br> • SVG<br> • PNG<br> • JPG<br><br> ✌️Author <br> DrawKit |
+| <a href="https://drawkit.com/product/covid-19-vaccinations"><img width='700' src='https://user-images.githubusercontent.com/34191321/194714608-5619de4e-2d0d-4023-ad47-95eb5be1d299.png' alt="COVID-19_&_Vaccinations_free_illustrations" /></a> | [COVID-19 & Vaccinations Illustrations](https://drawkit.com/product/covid-19-vaccinations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • SVG<br> • PNG<br> • AI<br><br> ✌️Author <br> DrawKit |
```

Comment:
- change the src link to `https://user-images.githubusercontent.com/74038190/195567351-f9f769ff-c308-4088-b71a-1617a14b030d.png` in covid-19-vaccinations